src/EditorFeatures/Core/ExternalAccess/VSTypeScript/VSTypeScriptNavigationBarItemService.cs

```diff
@@ -18,7 +18,7 @@
 
 namespace Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript
 {
-    [ExportLanguageService(typeof(INavigationBarItemServiceRenameOnceTypeScriptMovesToExternalAccess), LanguageNames.FSharp), Shared]
+    [ExportLanguageService(typeof(INavigationBarItemServiceRenameOnceTypeScriptMovesToExternalAccess), InternalLanguageNames.TypeScript), Shared]
```

Comment:
Anything we can do to prevent this mistake? It seems we made LanguageNames.FSharp public (!!!) but it'd be awesome if we could move the constant to only the FSharp external access layer somehow.

Reply:
> Anything we can do to prevent this mistake?

tests seem the sanest mechanism.  for example, we could ahve just forgotten this attribute in the first place.